reve/reve/lib/ModuleSimplifier.cpp

```diff
@@ -256,14 +256,19 @@ bool ModuleSimplifier::trySwap(FunctionAbstractionsGenerator::FunMap &Map,
     return false;
 }
 
-void ModuleSimplifier::runIndependentPasses(llvm::Module &Module) {
+void ModuleSimplifier::runIndependentPasses(llvm::Module &Module, llvm::Function& Main) {
```

Comment:
Space goes between type and & (please re-check everywhere, even in templates). Also no space after &. Same applies to *.